lib/Toolbar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import icons from './icons';
+
+class Toolbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { showHeaders: false };
+  }
+
+  render() {
+    return (<div id="quill-toolbar">
+      <div className="ql-formats">
+        <button
+          className="ql-show-headers"
+          onClick={() => this.setState({ showHeaders: !this.state.showHeaders })}
+        >
+          { icons.aa() }
```

Comment:
Just curious, what does `aa` mean?

Reply:
The aa is for this first icon, for changing between p, H1, .. H6 <img width="267" alt="screen shot 2017-08-25 at 17 16 10" src="https://user-images.githubusercontent.com/3472717/29722537-2307a6ee-89b9-11e7-9e0a-263e05d864a3.png">